src/hooks/sse/usePlaylistSSE.ts

```
@@ -0,0 +1,147 @@
+      eventSourceRef.current.addEventListener('connect', (message: MessageEvent) => {
+        dispatch({ type: 'SET_LIVE', live: true });
+        const data = JSON.parse(message.data);
+        dispatch({ type: 'SET_LISTENER', listenerCount: data.clientCount });
+        reconnectAttemptRef.current = 0;
+      });
+
+      eventSourceRef.current.addEventListener('video', (message: MessageEvent) => {
+        const data = JSON.parse(message.data);
+
+        if (data.status === 'DELETE') {
+          queryClient.setQueryData(['playlist', playlistCode], (oldData: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)